pkgdb2/lib/__init__.py

```diff
@@ -1529,6 +1644,238 @@ def add_branch(session, clt_from, clt_to, user):
     return messages
 
 
+def add_new_branch_request(session, pkg_name, clt_from, clt_to, user):
+    """ Register a new branch request.
+
+    This method only flushes the new objects.
```

Comment:
On Wed, Jan 21, 2015 at 07:54:15AM -0800, Ralph Bean wrote:
>    In pkgdb2/lib/__init__.py:
> 
>  > @@ -1529,6 +1644,238 @@ def add_branch(session, clt_from, clt_to, user):
>  >      return messages
>  > 
>  > 
>  > +def add_new_branch_request(session, pkg_name, clt_from, clt_to, user):
>  > +    """ Register a new branch request.
>  > +
>  > +    This method only flushes the new objects.
> 
>    This part of the docstring doesn't make sense. Could you elaborate more?

Actually it can be removed as it does not even ``session.flush()``, all it does
it ``session.add``. The idea was that there is nothing committed to the database
after this method has been called.